packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -7,29 +7,1024 @@ category: management
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+This request requires the `method` property as its request body. An optional boolean `retry` property can be used to indicate resend challenge when its value is set to `true`.
+
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| `method`  | String | The method with which the challenge should be sent, valid values are `SMS` and `CALL` |
+| `retry`  | Boolean | An optional property that indicates whether this is a normal challenge or retry |
+
+#### Response body
+
+Returns a JSON object containing a link to verify the challenged phone, with an HTTP 200 status code.
+
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| `_links` | Object ([JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06))  | Discoverable resources related to the caller's email |
+
+Passing an invalid `method` returns a 400 BAD REQUEST with error code E0000001.
+
+Disabling the factor type of the corresponding method on the org returns a 403 FORBIDDEN with error code E0000038.
+
+Passing an invalid `id` returns a 404 NOT FOUND with error code E0000008.
+
+Call providers failing to send the out of band OTP challenge returns 500 with error code E0000138.
+
+#### Usage example
+
+##### Request
+
+This request sends a verification code by SMS to the phone number represented by `id`. The request is a normal phone challenge, not a retry.
+
+```bash
+curl -XPOST 'https://${yourOktaDomain}/myaccount/phones/{id}/challenge' -H 'Authorization: bearer {token}' -H 'Content-Type: application/json' --data '{
+     "method": "SMS",
+     "retry": false
+ }'
+```
+
+##### Response
+
+Returns an HTTP 200 status code response.
+
+```bash
+{
+    "_links": {
+        "verify": {
+            "href": 'http://${yourOktaDomain}/idp/myaccount/phones/smsa7gpDMs5V6C0LT0g4/verify',
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Verify My Phone
+
+<ApiOperation method="post" url="/idp/myaccount/phones/{id}/verify"/>
+
+Verify the phone number with the verification code that the user receives via `SMS` or `CALL`. The phone number is active upon a successful verification.
+
+> **Note:** Sending requests to the `/idp/myaccount/phones/{id}/verify` endpoint at a rate that exceeds the rate limit rule configured by the admin, returns a 429 (Too Many Requests) error.
+
+#### Required scope and role
+
+An Okta scope of `okta.myAccount.phone.manage` is required to use this endpoint.
+
+> **Note:** Admin users are not allowed to call the `/idp/myaccount/phones/{id}/verify` endpoint.
+
+#### Required scope and role
```

Comment:
same here this Required scope and role seems to be a duplicate we should remove this 
```suggestion
```